verification/src/compiler/download_cache.rs

```diff
@@ -47,7 +49,8 @@ impl DownloadCache {
             }
             None => {
                 let _timer = metrics::COMPILER_FETCH_TIME.start_timer();
-                self.fetch(fetcher, ver).await
+                let span = tracing::debug_span!("fetch compiler", ver = ver.to_string());
+                self.fetch(fetcher, ver).instrument(span).await
```

Comment:
it's fine as of now, but just a question
I saw there is a `#instrument` macro, which will automatically add the span. Is there something like it for the trait function? Not the impl block, but the trait, like
```rust
trait Foo {
  #[instrument]
  fn foo();
}
```
If it exists, then I suggest changing the span to such macro, If not, current impl is okay

Reply:
no, I think it's not possible to add instrument to trait function